Docs/Configuration.tex

```diff
@@ -558,11 +558,11 @@ \subsection{Directory Structure}\label{directory-structure}
 
 \emph{Note}: It is not guaranteed that paths longer than
 \texttt{OC\_STORAGE\_SAFE\_PATH\_MAX} (128 characters including
-\texttt{\\0}-terminator) will be accessible within OpenCore.
+the null-terminator) will be accessible within OpenCore.
```

Comment:
It is not null terminator. Either nul, with one l, or \0. Better leave unchanged in my opinion.

Reply:
Changed

Reply:
For me it still shows broken.

Reply:
Meant to be changed and pushed.